lusine.sargsyan/repo/Lesson_3/exercise_17/exercise_17.py

```diff
@@ -0,0 +1,33 @@
+
+#! /usr/bin/env python3 
+
+number1, number2, number3 = input("enter three numbers: ") .split()
+
+print("Sum is: ", int(number1) + int(number2) + int(number3))
+
+print("Average is: ", round((int(number1) + int(number2) + int(number3)) / 3))
+
+print("Product is: ", int(number1) * int(number2) * int(number3))
+
+
+#the smallest 
+
+if number1 < number2:
```

Comment:
LW: Use fewer comparisions.

Hint. It is possible to write just with two if statements